AirCasting/Models/SessionExtension.swift

```diff
@@ -7,49 +7,49 @@
 
 import Foundation
 
-extension Session {
+extension SessionEntity {
     
-    var pm1Stream: MeasurementStream? {
+    var pm1Stream: MeasurementStreamEntity? {
         let matach = measurementStreams?.first(where: { (stream) -> Bool in
-            (stream as? MeasurementStream)?.sensorName?.contains("PM1") ?? false
+            (stream as? MeasurementStreamEntity)?.sensorName?.contains("PM1") ?? false
```

Comment:
I believe we had an error here and @anna1901 was fixing it. The error" "PM10" measurement also contains "PM1", and there was data from "PM10" displayed in both places. 
Not sure if it got lost or Ania didn't commit it yet.

Reply:
I think this is still not merged.

Reply:
Yes, it's not merged yet, cause it's in my mobile session branch.